scss/_footer.scss

```diff
@@ -1,23 +1,38 @@
 #footer_block_image_banner_outside {
-  background: no-repeat top left url(/images/illustrations/footer-planet.svg)
+  background: no-repeat top left url(/images/illustrations/footer-planet.svg);
 }
 
 #footer_block_image_banner_outside2 {
-  background: no-repeat bottom right url(/images/illustrations/footer-orange.svg)
+  background: no-repeat bottom right
+    url(/images/illustrations/footer-orange.svg);
 }
 
 #footer_block_image_banner_inside {
-//  @media only screen and (max-width: 100rem) {margin-left:140px;}
+  //  @media only screen and (max-width: 100rem) {margin-left:140px;}
 }
 
 .tags_links {
-    a {
-        margin-bottom: $inline-list-default-float-list-margin !important;
-    }
+  a {
+    margin-bottom: $inline-list-default-float-list-margin !important;
+  }
 }
 
 #footer_block {
-    ul {
-        display: inline-block;
-    }
+  ul {
+    display: inline-block;
+  }
+}
+
+@media #{$small-only} {
+  .inline-list {
+    margin-left: 0;
+  }
+  .title-5 {
+    font-size: 1.2857142857rem;
```

Comment:
Why is this font-size so specific? Have you taken it from somewhere?

Reply:
It's already there when I am testing in developer mode in chrome. I should have removed it.